content/python/concepts/tuples/tuples.md

```diff
@@ -1,74 +1,59 @@
 Subjects:
   - 'Computer Science'
   - 'Data Science'
 Tags:
+  - 'Collections'
   - 'Tuples'
   - 'Lists'
   - 'Data Types'
+  - 'Data Structures'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-A tuple is a Python data structure used to store an ordered and immutable collection of objects. Tuples are similar to lists, except that tuples cannot be changed after creation.
+A **tuple** is a data structure used to store an ordered and immutable collection of objects, unlike [dictionaries](https://www.codecademy.com/resources/docs/python/dictionaries) which are unordered and mutable. They are similar to [lists](https://www.codecademy.com/resources/docs/python/lists), except that tuples cannot be changed after creation.
 
-## Creating a Tuple
+Tuples also support built-in sequence functions such as [`len()`](https://www.codecademy.com/resources/docs/python/built-in-functions/len), [`min()`](https://www.codecademy.com/resources/docs/python/built-in-functions/min), [`max()`](https://www.codecademy.com/resources/docs/python/built-in-functions/max).
 
-There are multiple ways to create a tuple in Python.
+## Syntax
 
-Tuples can be created with the built-in `tuple()` constructor:
+```pseudo
+# With built-in function
+tuple(value1, value2, ... valueN)
 
-```py
-streaming_platforms = tuple(["Netflix", "Hulu", "Amazon", "Apple TV"])
-```
+# With parentheses (multi-item)
+tuple = (value1, value2, ... valueN)
 
-We can also create tuples with parentheses:
-
-```py
-mountains = ("Everest", "Kilimanjaro", "Fuji")
+# With parentheses (single item with trailing comma)
+tuple = (singleValue, )
 ```
 
-One-element tuples are a special case. We can create a one-element tuple with a trailing comma:
+There are multiple ways to create tuples in Python.
 
-```py
-controller = ("joystick", )
-```
+- The first way is with the built-in `tuple()` function that accepts an iterable value such as a list.
+- Tuples can also be created with just parentheses.
 
-Without the trailing comma, Python will interpet the surrounding parentheses as an expression instead of a tuple:
+**Note**: If the tuple is made with just parentheses and has only one element, it must contain a trailing comma. Otherwise, Python will interpet the surrounding parentheses as an expression instead of a tuple:
 
 ```py
 streaming_days = (3)
 
-print(type(streaming_days))
-# Output: <class 'int'>
+print(f"Value is {streaming_days}. Type is {type(streaming_days)}")
+# Output: Value is 3. Type is <class 'int'>
 ```
 
-## Using a Tuple
+## Codebyte Example
 
-Much like lists, since tuples are ordered, we can access tuple elements by index:
+The following example shows how tuples are like lists in that they are ordered and their elements can be accessed by index. However, since tuples are immutable, we cannot modify, delete, or add new elements. Attempting to re-assign an element will throw a `TypeError`:
```

Comment:
Should we open an issue to add `TypeError` to [Python: Errors](https://www.codecademy.com/resources/docs/python/errors) entry?

Reply:
[Filed](https://github.com/Codecademy/docs/issues/856)!